src/switch_rtp.c

```diff
@@ -5130,6 +5132,16 @@ SWITCH_DECLARE(switch_status_t) switch_rtp_create(switch_rtp_t **new_rtp_session
 		}
 	}
 
+	rtp_session->poll_timeout_s = TELNYX_RTP_DEFAULT_POLL_TIMEOUT_S;
```

Comment:
There are issues now, that's why we want to change this to 2 by default.

Reply:
This also makes little sense to have this setting in global config (btw: there is no file switch.conf, you probably think about switch.conf.xml), becuase:
- default setting (of 2 seconds) will work by default, if you need to have specific settings for given connection, you will configure that in Dialplan (or user hacks), note **per number/connection**, so it must be read from session
- you will not have to change it, by default, if you do, you can compile new deb pkg

I will put that to switch.conf.xml if you still believe it may be needed though.